src/popup/components/AccountDetailsBase.vue

```diff
@@ -277,6 +267,7 @@ export default defineComponent({
   .tabs-content {
     position: relative;
     padding: 0 var(--screen-padding-x);
+    height: 350px;
```

Comment:
Please avoid using magic numbers. This `350` gives no context. What if the app will have different size in the future?